src/django/api/serializers/activity_log.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework.serializers import (
+    ModelSerializer,
+    SerializerMethodField,
+    EmailField,
+    DateTimeField,
+)
+
+from ..models.user import User
+from ..models.submission import Submission
+
+
+class ActivityLogSerializer(ModelSerializer):
+    user = EmailField(source='created_by.email')
+    action = SerializerMethodField()
+    time = DateTimeField(source='created_at')
+
+    def get_action(self, value):
+        return 'Drafted'
+
+    class Meta:
+        model = Submission
+        fields = ['user', 'action', 'time', 'notes']
+        optional_fields = ['annotations_count']
+
+    def to_representation(self, instance):
+        obj = super(ActivityLogSerializer, self).to_representation(instance)
+        if instance.submitted_at is not None:
+            submitter = User.objects.get(pk=instance.submitted_by.pk)
+            obj['action'] = 'Submitted'
+            obj['time'] = instance.submitted_at
+            obj['user'] = submitter.email
+        if hasattr(instance, 'review'):
+            review = instance.review
+            reviewer = User.objects.get(pk=review.reviewed_by.pk)
+            if review.reviewed_at:
+                obj['action'] = 'Reviewed'
+                obj['time'] = review.reviewed_at
+                obj['user'] = reviewer.email
+                obj['notes'] = review.notes
+                obj['annotations_count'] = review.annotations.count()
+        if hasattr(instance, 'approval'):
+            approval = instance.approval
+            approver = User.objects.get(pk=approval.approved_by.pk)
+            if approval.approved_at:
+                obj['action'] = 'Approved'
+                obj['time'] = approval.approved_at
+                obj['user'] = approver.email
+        return obj
```

Comment:
I noticed reviews and approvals are blocking activity entries for submissions/drafts. For instance, if you visit http://localhost:8181/api/boundaries/5/, you get a single entry for an approval. I would have expected to have also seen the drafted, submitted, etc.

We can exchange some solutions over Slack?